travis.gemspec

```diff
@@ -368,17 +369,13 @@ Gem::Specification.new do |s|
   ]
 
   # dependencies
-  s.add_dependency "faraday",               "~> 0.9", "< 0.17.1" if RUBY_VERSION < "2.0"
-  s.add_dependency "faraday",               "~> 0.9" if RUBY_VERSION >= "2.0"
+  s.add_dependency "faraday",               "~> 0.9"
```

Comment:
Could you update to 1.0 here? (#736)
Without that it's hard to use this gem efficiently as it basically enforces sequential blocking requests (https://github.com/socketry/async-http-faraday/issues/16)

Reply:
As I stated in that ticket, it will be. :-)